components/Header.vue

```diff
@@ -1,8 +1,13 @@
 <template>
   <header class="w-full p-4 shadow-md sticky top-0 bg-surface-primary z-10">
-    <div class="desktop-gutters flex items-center justify-center space-x-2">
-      <icons-centrapay-logo class="w-12 h-12 fill-brand-accent" />
-      <span class="hidden md:block type-headline-3">Centrapay Docs</span>
+    <div class="flex items-center justify-start space-x-2">
+      <NuxtLink
+        to="/"
+        class="flex items-center"
+      >
+        <icons-centrapay-logo class="w-10 h-10 fill-brand-accent" />
+        <span class="hidden md:block type-headline-4 text-content-primary px-1">Centrapay Docs</span>
```

Comment:
Can you please change this HTML tag to be <h1> ?
It is the main text for the heading and this makes a difference when people use screen readers 👍

Reply:
The class `text-content-primary` could also be removed as this is already in the definition for `type-headline-4` ☺️
You can see this in the file `assets/css/plugins/typography.js`.